assignments/hackyourtemperature/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "hackyourtemperature",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": { "start": "nodemon server.js"
+    
+  },
```

One suggestion to keep the code consistent in its format

```suggestion
  "scripts": {
    "start": "nodemon server.js"    
  },
```